crates/bevy_render/src/renderer/mod.rs

```diff
@@ -88,6 +88,10 @@ pub fn render_system(world: &mut World) {
 #[derive(Resource, Clone, Deref, DerefMut)]
 pub struct RenderQueue(pub Arc<Queue>);
 
+/// This is the adapter that the GPU is using to render.
```

Comment:
This doc comment isn't helpful to someone who doesn't know what an adapter is.

We should consider linking to the `wgpu` docs here.

Reply:
Done by following @devil-ira's suggestion.